### PR TITLE
Add notes about contributions and Sociomantic Tsunami affiliation to README

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
- Mention affiliation with Sociomantic Tsunami
- Minor formatting fixes
- Add note about contributing to the library

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,14 +5,16 @@ Description |BuildStatus|_
 C connector. It is designed to allow you to connect to and query a
 **MySQL** database server using a simple API.
 
-Libdrizzle-redux in the sociomantic tsunami repository is a fork of the official
-**v5.1.4 Felmingham** at https://launchpad.net/libdrizzle.
+**Libdrizzle-redux** is a fork of the official **v5.1.4 Felmingham** release available at https://launchpad.net/libdrizzle.
```
Comment (mathias-lang-sociomantic) [inline README.rst]: Is it the last release available ?

File: README.rst [hunk 2]
```diff
@@ -5,14 +5,16 @@ Description |BuildStatus|_
...
 
 The primary reason for the fork is that development of the upstream stopped in 2013
-which meant that crucial fixes and enhancements  were not included in the latest
+which meant that crucial fixes and enhancements were not included in the latest
```
Comment (mathias-lang-sociomantic) [inline README.rst]: I'd just say that development stopped. Fixes and enhancements not being included is a bit of a given in this situation. But no strong opinion

File: README.rst [hunk 3]
```diff
@@ -5,14 +5,16 @@ Description |BuildStatus|_
...
 stable release **v5.1.4 Felmingham**.
 
-We hope that continuing the development of libdrizzle-redux can benefit current
+The repository was released under the `Tsunami programme`_ created by `Sociomantic Labs`_
```
Comment (mathias-lang-sociomantic) [inline README.rst]: was ?
Proposition: "This repository is part of..."
IIRC `sociomantic labs GmbH` is how legal wants us to refer to it. Check swarm headers for confirmation.

File: README.rst [hunk 4]
```diff
@@ -114,6 +116,11 @@ Documentation can be generated in several formats::
     epub : make epub
     html : make html
 
+Contributing to libdrizzle-redux
+--------------------------------
+
+Please refer to the guide for users of libraries at `Neptune Wiki`_.
```
Comment (mathias-lang-sociomantic) [inline README.rst]: This is a bit thrown in the middle. What's Neptune about ? I am wondering where a function is used, is that where I should go ?

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: Yes, v5.1.5 wasn't released before dev on the the original project stopped, https://launchpad.net/libdrizzle/5.1

File: README.rst (hunk 2 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: I'd keep it. We did add the mentioned enhancements and fixes which weren't released after Felmingham

File: README.rst (hunk 3 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: Genau

File: README.rst (hunk 4 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: So you mean that 

> What's Neptune about ? I am wondering where a function is used, is that where I should go ?

Should be the body of the section, followed by a link to neptune?

Comment (ben-palmer-sociomantic) [reply README.rst]: I think he just means that there isn't too much explanation of what Neptune is. You could do a brief introduction sentence on Neptune might be more clear.

File: README.rst (hunk 2 above)
Comment (ben-palmer-sociomantic) [reply README.rst]: I'd slightly side with Mathias on this one but not strong feelings.

**Review by mathias-lang-sociomantic — APPROVED** [review]

**Pull request merged**